docs/getting-started.md

```diff
@@ -780,10 +780,13 @@ This can be useful for debugging a failing extractor or loader.
 Now that you've successfully run your first data integration (EL) pipeline using Meltano,
 you have a few possible next steps:
 
-- [Schedule pipelines to run regularly](#schedule-pipelines-to-run-regularly)
-- [Transform loaded data for analysis](#transform-loaded-data-for-analysis)
-- [Containerize your project](#containerize-your-project)
-- [Deploy your pipelines in production](#deploy-your-pipelines-in-production)
+- [Example](#example)
```

Comment:
What's the purpose of nesting these this way?

Reply:
There is no purpose. Annoyingly, this change was made without my knowledge by the VS Code Markdown extension. It thought that the list was a table of content, and so it automatically reformatted it. I have disabled the `toc.updateOnSave` setting, and now it has been fixed. Good catch.

Reply:
Haha well damn, I was hoping there was a cool secret here but that works too. Thanks!